src/fides/api/schemas/tcf.py

```diff
@@ -14,40 +14,50 @@
 from fides.api.schemas.privacy_notice import UserSpecificConsentDetails
 
 
-class TCFSavedandServedDetails(UserSpecificConsentDetails):
-    """Default Schema that combines TCF details with whether a consent item was
-    previously saved or served."""
+class EmbeddedVendor(FidesSchema):
+    """Sparse details for an embedded vendor beneath a purpose or feature section. Read-only."""
+
+    id: str
+    name: str
+
+
+class NonVendorSection(UserSpecificConsentDetails):
+    """Common details for non-vendor TCF sections.  Includes previously-saved preferences and
+    records where consent was previously served if applicable."""
+
+    vendors: List[EmbeddedVendor] = []  # Vendors that use this TCF attribute
+    systems: List[EmbeddedVendor] = []  # Systems that use this TCF attribute
+
+
+class TCFPurposeConsentRecord(NonVendorSection, MappedPurpose):
+    """Schema for a TCF Purpose with Consent Legal Basis returned in the TCF Overlay Experience"""
 
     @root_validator
     def add_default_preference(cls, values: Dict[str, Any]) -> Dict[str, Any]:
-        """For TCF components, the default preferences are just 'opt-out'"""
+        """Default preference for purposes with legal basis of consent is opt-out"""
         values["default_preference"] = UserConsentPreference.opt_out
 
         return values
 
-    class Config:
-        use_enum_values = True
 
+class TCFPurposeLegitimateInterestsRecord(NonVendorSection, MappedPurpose):
+    """Schema for a TCF Purpose with Legitimate Interests Legal Basis returned in the TCF Overlay Experience"""
 
-class EmbeddedVendor(FidesSchema):
-    """Sparse details for an embedded vendor beneath a purpose or feature section. Read-only."""
+    @root_validator
+    def add_default_preference(cls, values: Dict[str, Any]) -> Dict[str, Any]:
+        """Default preference for purposes with legal basis of legitimate interests is opt-in"""
+        values["default_preference"] = UserConsentPreference.opt_in
 
-    id: str
-    name: str
+        return values
 
 
-class TCFPurposeRecord(MappedPurpose, TCFSavedandServedDetails):
-    """Schema for a TCF Purpose or a Special Purpose: returned in the TCF Overlay Experience"""
+class TCFSpecialPurposeRecord(NonVendorSection, MappedPurpose):
+    @root_validator
+    def add_default_preference(cls, values: Dict[str, Any]) -> Dict[str, Any]:
+        """Default preference for special purposes is acknowledge"""
+        values["default_preference"] = UserConsentPreference.acknowledge
```

Comment:
oh you already did it!!!! okay that's half of https://github.com/ethyca/fides/issues/4141 done 💥